wonjong/models/userDao.js

```diff
@@ -39,10 +39,33 @@ const userAllPostView = async (userId) => {
         JOIN users u ON p.user_id = u.id
         WHERE p.user_id = ?
         GROUP BY p.user_id;`,
-      { userId }
+      [userId]
+    );
+    return user;
+  } catch (err) {
+    const error = new Error('DO_NOT_GET_DATA');
+    console.log(err);
+    error.statusCode = 400;
+    throw error;
+  }
+};
+
+const loginCheckPassword = async (email) => {
```

Comment:
함수명을 정할 때 해당 함수가 어떤 행위를 하는지 명확히 나타내는 것이 좋습니다. 
해당 함수는 SQL을 통해 user 정보를 가지고 오고 있고, 가지고 올 때 조건이 명확하게 email이므로 함수 이름을 다음과 같이 바꾸면 좋을 것 같습니다.
```suggestion
const getUserByEmail = async (email) => {
```